docker-app/qfieldcloud/core/management/commands/dequeue.py

```diff
@@ -101,13 +101,6 @@ def handle(self, *args, **options):
             if options["single_shot"]:
                 break
 
-    def run(self, job_id, *args, **options):
-        try:
-            job = Job.objects.get(id=job_id)
-            self._run(job)
-        except Exception:
-            pass
-
```

Comment:
Seems dead code. And because there is also a JobRunner.run() it causes quite a bit of confusion.

Reply:
Agreed. Thanks for spotting.